src/IndividualReview.jsx

```diff
@@ -1,24 +1,29 @@
 import { useEffect, useState } from "react";
-import axios from "axios";
-import { useLocation, useParams } from "react-router";
+import { useParams } from "react-router";
 import { Link } from "react-router-dom";
+import { getReviewById } from "./Api";
+import { formatDate } from "./utilities";
 
 const IndividualReview = () => {
   let { reviewId } = useParams();
-  const [review, setReview] = useState("");
+  const [review, setReview] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [date, setDate] = useState("");
 
   useEffect(() => {
-    axios
-      .get(
-        `https://fame-boardgame-review-website.onrender.com/api/reviews/${reviewId}`
-      )
-      .then(({ data }) => {
-        setReview(data.review);
-      });
+    setIsLoading(true);
+    getReviewById({ setReview, reviewId });
+    setIsLoading(false);
```

Comment:
This setIsLoading to false should only happen if our request was successful (i.e. in a .then block)